src/pages/documentation/platform-assurance/tutorials/event-transactions.md

```diff
@@ -0,0 +1,61 @@
+# Event Transactions view
+
+## Overview of Event Transactions view in Assurance
+
+The Event Transactions view in Adobe Experience Platform Assurance enables you to validate and debug your Edge Network client implementation, and see the upstream validation results in near real time.
+
+## Set up Assurance for Edge Network
+
+To get started, complete the following steps:
+
+1. [Set up Assurance](../set-up.md).
+2. Ensure that you have implemented the latest versions of the Assurance and Edge Network extensions in your app.
+3. To view your events, from the left side menu select the **Event Transactions** under the **ADOBE EXPERIENCE PLATFORM EDGE** section.
+  If you do not see this option, select **Configure** from the bottom left of the window, add the **Event Transactions** view and select **Save**.
+
+## Get started using the view
+
+In this section, get familiar with the Event Transaction view and learn how to use it efficiently for end to end validation on Edge Network workflows.
+
+### Event processing flow
+
+![Event transactions view](./assets/event-transactions/event-transactions-view.png)
+
+At a high-level the Event Transactions view displays 3 columns in the order of the event processing flow:
+
+- **CLIENT-SIDE**  - This column represents the events processed or received client-side, accessible to the Mobile SDK. This includes the events that were collected based on an API call, such as `Edge.sendEvent`, or the response handles received by the client from the Edge Network server. Examples of client-side events:
+  - AEP Request Event represents the event sent through the Edge extension and contains the XDM and free-form data.
+  - AEP Response Event Handle represents the event handle received from Edge Network in response to an AEP Request Event. An request event may receive none, one or multiple response event handles.
+  - AEP Error Response may be seen in case of an error, for example if the XDM payload could not be processed or if one of the upstream services returned an error or warning.
```

Comment:
Do the names actually contain AEP in it? Since that's against branding, so I'm a _little_ annoyed that, if it's an official name, that it contains AEP in it...

Reply:
yes, that is the name of at this time